stripe/api_resources/terminal/reader.py

```diff
@@ -53,7 +53,6 @@ class TestHelpers(APIResourceTestHelpers):
         def present_payment_method(self, idempotency_key=None, **params):
             url = self.instance_url() + "/present_payment_method"
             headers = util.populate_headers(idempotency_key)
-            self.resource.refresh_from(
-                self.resource.request("post", url, params, headers)
-            )
-            return self.resource
+            resp = self.resource.request("post", url, params, headers)
```

Comment:
This might be a breaking change, we are not refreshing the original resource anymore..